src/instructlab/eval/mt_bench.py

```diff
@@ -94,26 +94,29 @@ class MTBenchEvaluator(AbstractMTBenchEvaluator):
 
     name = "mt_bench"
 
-    def gen_answers(self, server_url) -> None:
+    def gen_answers(self, api_key, server_url) -> None:
         """
         Asks questions to model
 
         Attributes
+            api_key         API token for authenticating with model server
             server_url      Model server endpoint (Ex: http://localhost:8000/v1) for the model being evaluated
         """
         logger.debug(locals())
         mt_bench_answers.generate_answers(
             self.model_name,
+            api_key,
             server_url,
             output_dir=self.output_dir,
             max_workers=self.max_workers,
```

Comment:
The api_key has been added as the third parameter, not the second, in `mt_bench_answers`. So I think this is actually setting the api_key to the server_url and vice_versa. 

```suggestion
           
            server_url,
            api_key=api_key
            output_dir=self.output_dir,
            max_workers=self.max_workers,
```